MTVd-test-project/index.html

```diff
@@ -0,0 +1,302 @@
+                            <div class="latest-news-item-column">
+                                <div class="circle-item">
+
+                                </div>
+                            </div>
+                            <div class="latest-news-item-column-text">
+                                <p class="item-header">
+                                        Three tricks to becoming a better writer
+                                </p>
+                                <p class="item-data">
+                                    14 February 2022
+                                </p>
+                                <p class="item-text">
+                                    Lorem ipsum dolor sit amet,
+                                    consectetur adipisicing elit,
+                                    sed do eiusmod tempor incididunt ut labore et dolore
+                                    magna aliqua. Ut enim ad minim veniam,
+                                    quis nostrud exercitation ullamco.
+                                </p>
+                            </div>
+                        </div>
+                        <div class="latest-news-content-item">
+                            <div class="latest-news-item-column">
+                                <div class="circle-item">
+
+                                </div>
+                            </div>
+                            <div class="latest-news-item-column-text">
+                                <p class="item-header">
+                                        Quality content is better than quantity of content
+                                </p>
+                                <p class="item-data">
+                                    14 February 2022
+                                </p>
+                                <p class="item-text">
+                                    Lorem ipsum dolor sit amet,
+                                    consectetur adipisicing elit,
+                                    sed do eiusmod tempor incididunt ut labore et dolore
+                                    magna aliqua. Ut enim ad minim veniam,
+                                    quis nostrud exercitation ullamco.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+        </div>
+        <!-- FOOTER BLOCK -->
+        <footer class="footer">
+                <div class="brands-row">
+                        <img src="../images/brand-item-first.png" class="brand-item"/>
+                        <img src="../images/brand-item-second.png" class="brand-item"/>
+                        <img src="../images/brand-item-third.png" class="brand-item"/>
+                        <img src="../images/brand-item-forth.png" class="brand-item"/>
+                </div>
+            <div class="footer-content">
+                <div class="footer-row-nav">
```

Comment:
Please use UL + LI for every footer-column (two ul+li combinations, one is to build a list of footer-column, and inside every item-column also a UL+LI instead of every P.